### PR TITLE
fichier en cours de modification

### PR DESCRIPTION
**Pull request by SkiveF**
il y aura surment d'autre modif à faire 
A voir...

### CODE REVIEW

File: backend-poll-generator/backend/workers/formol_eventstream.js [hunk 1]
```diff
@@ -18,3 +18,18 @@ const tasks = require('../tasks');
 // );
 
 // module.exports = worker;
+const worker = new Worker("queueName",
+    {connection:redisConn},
+    async job =>{
+        if(job.ID === "12test34"){
+            worker.on("completed", (jobId)=>{
```
Comment (Themimitoof) [inline backend-poll-generator/backend/workers/formol_eventstream.js]: This block and the block in L29 are not needed. You can that empty for now or with a cute little message.

File: backend-poll-generator/backend/workers/formol_eventstream.js [hunk 2]
```diff
@@ -18,3 +18,18 @@ const tasks = require('../tasks');
...
+                console.log(`Job ${jobId}, is running.....`);
+            });
+        }else{
+          worker.on("failed", (jobId, err)=>{
+            console.log(`${jobId} has failed with ${err.message}`);
+          });
+        }
+    }
+);
+module.exports = worker;
```
Comment (Themimitoof) [inline backend-poll-generator/backend/workers/formol_eventstream.js]: It seems you've not read the [CONTRIBUTION.md](CONTRIBUTION.md) file about prettier and editorconfig, missing blank line at the end of the file ;)

**Review by Themimitoof — CHANGES_REQUESTED** [review]

File: backend-poll-generator/backend/workers/formol_eventstream.js (hunk 1 above)
Comment (SkiveF) [reply backend-poll-generator/backend/workers/formol_eventstream.js]: ok understood !!!

File: backend-poll-generator/backend/workers/formol_eventstream.js (hunk 2 above)
Comment (SkiveF) [reply backend-poll-generator/backend/workers/formol_eventstream.js]: You have a little bit reason lol
i didn't read all

**Pull request closed without merging**